spectral_cube/io/casa_image.py

```diff
@@ -111,11 +113,12 @@ def load_casa_image(filename, skipdata=False,
 
     # read in the data
     if not skipdata:
-        data = ia.getchunk().reshape(ia.shape())
+        # CASA data are apparently transposed.
+        data = ia.getchunk().transpose()
```

Comment:
I'm wondering if a non-standard stokes axis screws this up somehow. Maybe we've already dealt with this for `StokesSpectralCube`. Just noting it here.

Reply:
Yeah, we had always had a `.transpose()` in stokes, but I think it turns out that CASA _always_ transposes the data, and we had never tested against stokesless cubes before (since CASA doesn't usually produce them...)

Reply:
(it was `.T`, not `.transpose()`, if you go hunting)

Reply:
Oh ok.